docs/workflow/building/coreclr/android.md

```diff
@@ -1,100 +1,172 @@
-Cross Compilation for Android on Linux
-======================================
+# Experimental support of CoreCLR on Android
 
-Through cross compilation, on Linux it is possible to build CoreCLR for arm64 Android.
+This is the internal documentation which outlines experimental support of CoreCLR on Android and includes instructions on how to:
+- [Build CoreCLR for Android](./android.md#building-coreclr-for-android)
+- [Build and run a sample application with CoreCLR](./android.md#building-and-running-a-sample-app)
+- [Debug the sample app and the runtime](./android.md#debugging-the-runtime-and-the-sample-app)
 
-Requirements
-------------
+## Prerequisite
 
-You'll need to generate a toolchain and a sysroot for Android. There's a script which takes care of the required steps.
+Download and install `Android Studio` and the following:
+  - Android SDK (minimum supported API level is 21)
+  - Android NDK r27
 
-Generating the rootfs
----------------------
+## Building CoreCLR for Android
 
-To generate the rootfs, run the following command in the `coreclr` folder:
+Supported host systems for building CoreCLR for Android:
+- [MacOS](./android.md#macos-and-linux) ✔
+- [Linux](./android.md#macos-and-linux) ✔
+- [Windows](./android.md#windows) ❌ (only through WSL)
+
+Supported target architectures:
+- x86 ❌
+- x64 ✔
+- arm ❌
+- arm64 ✔
+
+### MacOS and Linux
+
+#### Requirements
+
+Set the following environment variables:
+  - ANDROID_SDK_ROOT=`<full-path-to-android-sdk>`
+  - ANDROID_NDK_ROOT=`<full-path-to-android-ndk>`
+
+#### Building the runtime, libraries and tools
+
+To build CoreCLR runtime, libraries and tools for local development, run the following command from `<repo-root>`:
 
 ```
-cross/init-android-rootfs.sh
+./build.sh clr.runtime+clr.alljits+clr.corelib+clr.nativecorelib+clr.tools+clr.packages+libs -os android -arch <x64|arm64> -c <Debug|Release>
```

Comment:
MacOS requires `-cross`

Reply:
Does it? I think this was fixed late in the first bring-up PR.

Reply:
It is not required anymore as per Steve's changes